### DISCUSSION TITLE
ctrl-c kills ssh session

### DISCUSSION THREAD

**Discussion by blakeduffey** [PowerShell/Win32-OpenSSH#2064] [General]
cross-posting from https://github.com/microsoft/terminal/issues/12431

This happens on versions of openssh in windows terminal, vanilla powershell, or cmd

ctrl-c simply kills the ssh session

**client_loop: send disconnect: Broken pipe**

**Comment by maertendMSFT** [comment]
This does not repro on Win32-OpenSSH.  Linking to your comment on the linked thread: https://github.com/microsoft/terminal/issues/12431#issuecomment-1543988903
